src/AspNet.Security.OAuth.Docusign/DocusignAuthenticationOptions.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.Docusign;
+
+/// <summary>
+/// Defines a set of options used by <see cref="DocusignAuthenticationHandler"/>.
+/// </summary>
+public class DocusignAuthenticationOptions : OAuthOptions
+{
+    /// <summary>
+    /// Sets or retrieves a value that determines whether development or production endpoints are used.
+    /// The default value of this property is <see cref="DocusignAuthenticationEnvironment.Development"/>.
+    /// </summary>
+    public DocusignAuthenticationEnvironment Environment
```

Comment:
This could be a simple property, then we can use a post configure options to assign the properties (or throw if it's not one of the two expected values) for the URIs if not explicitly set. I forget which providers but there are existing examples of this pattern in the codebase.